js/app.js

```diff
@@ -17,35 +17,77 @@ var resultsList = document.createElement('ul');
 var names = [];
 var shown = [];
 var clicks = [];
-var productState = null;
+var productState = getProductState();
 var storageProductState;
 
 
-// if there's data in storage, pull it out
-if (!productState) {
-  productState = getProductState();
+// ================================
+// ===========MAIN=================
+// ================================
+// Main function to first check if there's data in storage; if there is, pull
+// it out and initialize the variables with the data in storage
+function main () {
+  if (!productState) {
```

Comment:
You need to remove the exclamation mark.

Right now, you're telling the program, "If local storage does not exist, load from local store.  If it does exist, use the default."

Which is the opposite of what you want.